### PR TITLE
Change DB API design

### PR DESCRIPTION
**Pull request by kumilange**
Since there seemed to be a problem with heroku deployment, I changed the db & api design to the newer one(referred to '`continuous_delivery_repo`').

1. Removed manual migration and rollback, run the migration on running the server `server/bin/www`
2. Deleted all the service dirs and moved knex handlings to `routes/api.js`
3. Moved db access methods to `util/helper.js`

### CODE REVIEW

File: server/utils/helper.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+// helper methods for db access
```
Comment (GameKyuubi) [inline server/utils/helper.js]: Change the name of this file to be more descriptive.  Perhaps `db-helper.js`

**Comment by GameKyuubi** [comment]
LGTM!
[LGTM](http://www.lgtm.in/i/MK1gZAXcq)

[:+1:](http://www.lgtm.in/u/MK1gZAXcq) [:-1:](http://www.lgtm.in/r/MK1gZAXcq)

**Review by GameKyuubi — DISMISSED** [review]
LGTM!
[LGTM](http://www.lgtm.in/i/MK1gZAXcq)

[:+1:](http://www.lgtm.in/u/MK1gZAXcq) [:-1:](http://www.lgtm.in/r/MK1gZAXcq)

File: server/utils/helper.js (hunk 1 above)
Comment (kumilange) [reply server/utils/helper.js]: Fixed it! :)

**Review by billfeng — APPROVED** [review]
LGTM

**Pull request merged**